<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignatureDialog</class>
 <widget class="QDialog" name="SignatureDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1014</width>
    <height>570</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1014</width>
    <height>516</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="containerLists" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>880</width>
       <height>399</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #ffffff;</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="leftPane" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>505</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>505</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-right: 1px solid #DEE4E9;</string>
        </property>
        <widget class="QLabel" name="lblSignerHeader">
         <property name="geometry">
          <rect>
           <x>21</x>
           <y>14</y>
           <width>278</width>
           <height>25</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>278</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>278</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Roboto</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: none;</string>
         </property>
         <property name="text">
          <string>Allkirjastaja</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="lblSigningCountry">
         <property name="geometry">
          <rect>
           <x>13</x>
           <y>122</y>
           <width>65</width>
           <height>16</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>65</width>
           <height>16</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>65</width>
           <height>16</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Roboto</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: none;</string>
         </property>
         <property name="text">
          <string>Riik</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="signerCity">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>45</y>
           <width>395</width>
           <height>31</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>395</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>395</width>
           <height>31</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 1px solid #DEE4E9;
border-radius: 2px;
background-color: #FFFFFF;</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLineEdit" name="signerState">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>80</y>
           <width>395</width>
           <height>31</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>395</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>395</width>
           <height>31</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 1px solid #DEE4E9;
border-radius: 2px;
background-color: #FFFFFF;</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLineEdit" name="signerCountry">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>115</y>
           <width>395</width>
           <height>31</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>395</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>395</width>
           <height>31</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 1px solid #DEE4E9;
border-radius: 2px;
background-color: #FFFFFF;</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLabel" name="lblSigningZipCode">
         <property name="geometry">
          <rect>
           <x>13</x>
           <y>157</y>
           <width>65</width>
           <height>16</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>65</width>
           <height>16</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>65</width>
           <height>16</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Roboto</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: none;</string>
         </property>
         <property name="text">
          <string>Postikood</string>
         </property>
        </widget>
        <widget class="QLabel" name="lblSigningCity">
         <property name="geometry">
          <rect>
           <x>13</x>
           <y>52</y>
           <width>65</width>
           <height>16</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>65</width>
           <height>16</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>65</width>
           <height>16</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Roboto</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: none;</string>
         </property>
         <property name="text">
          <string>Linn</string>
         </property>
        </widget>
        <widget class="QLabel" name="lblSigningCounty">
         <property name="geometry">
          <rect>
           <x>13</x>
           <y>87</y>
           <width>65</width>
           <height>16</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>65</width>
           <height>16</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>65</width>
           <height>16</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Roboto</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: none;</string>
         </property>
         <property name="text">
          <string>Maakond</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="signerZip">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>150</y>
           <width>395</width>
           <height>31</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>395</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>395</width>
           <height>31</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 1px solid #DEE4E9;
border-radius: 2px;
background-color: #FFFFFF;</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLabel" name="lblRole">
         <property name="geometry">
          <rect>
           <x>13</x>
           <y>200</y>
           <width>140</width>
           <height>16</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>16</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>140</width>
           <height>16</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Roboto</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: none;</string>
         </property>
         <property name="text">
          <string>Roll / resolutsioon</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="signerRoles">
         <property name="geometry">
          <rect>
           <x>13</x>
           <y>220</y>
           <width>472</width>
           <height>31</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>472</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>472</width>
           <height>31</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 1px solid #DEE4E9;
border-radius: 2px;
background-color: #FFFFFF;</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="AccordionTitle" name="showErrors" native="true">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>340</y>
           <width>485</width>
           <height>40</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">border: none;</string>
         </property>
        </widget>
        <widget class="QTextEdit" name="error">
         <property name="geometry">
          <rect>
           <x>13</x>
           <y>380</y>
           <width>472</width>
           <height>135</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 1px solid #DEE4E9;
border-radius: 2px;
background-color: #FFFFFF;</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLabel" name="info">
         <property name="geometry">
          <rect>
           <x>13</x>
           <y>280</y>
           <width>472</width>
           <height>60</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border: none;</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="openExternalLinks">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLabel" name="lblNotice">
         <property name="geometry">
          <rect>
           <x>13</x>
           <y>260</y>
           <width>140</width>
           <height>16</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>16</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>140</width>
           <height>16</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Roboto</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: none;</string>
         </property>
         <property name="text">
          <string>Notice</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="rightPane" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>9</number>
         </property>
         <property name="rightMargin">
          <number>9</number>
         </property>
         <property name="bottomMargin">
          <number>15</number>
         </property>
         <item>
          <widget class="QLabel" name="lblSignatureHeader">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>490</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>21</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Roboto</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border: none;</string>
           </property>
           <property name="text">
            <string>Allkiri</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTreeWidget" name="signatureView">
           <property name="font">
            <font>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border: none;</string>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="columnCount">
            <number>2</number>
           </property>
           <column>
            <property name="text">
             <string notr="true">Attribute</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string notr="true">Value</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="navigationArea" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-top: 1px solid #DEE4E9;
background-color: #f7f7f7;</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>13</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="title">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>750</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>100</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="close">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	border-radius: 2px; 
	border: none;
	color: #ffffff;
	background-color: #006EB5;
}
QPushButton:pressed {
	background-color: #41B6E6;
}
QPushButton:hover:!pressed {
	background-color: #008DCF;
}
QPushButton:disabled {
	background-color: #BEDBED;
}</string>
        </property>
        <property name="text">
         <string>SULGE</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Preferred</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>30</width>
          <height>13</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AccordionTitle</class>
   <extends>QWidget</extends>
   <header>widgets/AccordionTitle.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
